meteor/client/lib/forms/schema-types.ts

```diff
@@ -0,0 +1,1528 @@
+/*
+ * BSD-2-Clause License
+ *
+ * Original source code is copyright (c) 2022 Jeremy Rylan
```

Comment:
Because the original uses a top level boolean which greatly confuses our typescript version: https://github.com/jrylan/json-schema-typed/blob/main/dist/node/draft-2020-12.ts#L56

Validation is a good idea in general although I don't believe we benefit from further typings inside TSR. We should perhaps add some follow up tasks as this task has started to grow over time so consider this to be the ground work.